liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/GenerateFeaturesMojo.java

```diff
@@ -327,24 +320,22 @@ private void generateFeatures() throws MojoExecutionException, PluginExecutionEx
         }
     }
 
-    // Get the features from the server config and optionally exclude the specified config files from the search.
-    private Set<String> getServerFeatures(ServerFeatureUtil servUtil, Set<String> generatedFiles, boolean excludeGenerated) {
+    // Get the features from the server config
+    private Set<String> getServerFeatures(ServerFeatureUtil servUtil) {
         servUtil.setLowerCaseFeatures(false);
-        // if optimizing, ignore generated files when passing in existing features to
-        // binary scanner
         FeaturesPlatforms fp = servUtil.getServerFeatures(configDirectory, serverXmlFile,
-                new HashMap<String, File>(), excludeGenerated ? generatedFiles : null); // pass generatedFiles to exclude them
+                new HashMap<String, File>(), null);
```

Comment:
Discussed with @turkeylurkey. Plan is to update this method to still use the `generatedFiles` and `excludeGenerated` params, but pass the `serverDirectory` instead of the `configDirectory`.

Reply:
Updated.